samples/Velocity Template Language/hello-world.vtl

```diff
@@ -0,0 +1,12 @@
+#**
+  Here's a short story about this document.
+    @author John Doe
+    @version 1
+*#
+
+<html>
+  <body>
+  #set( $foo = "Hello" )
+   $foo World! ## This would print, you guessed it, Hello World!
+  </body>
+</html>
```

Comment:
Please remove/replace this sample. We explicitly ask for real-world samples as "hello worlds" are never really not generally illustrative of a language